grpc/proto/th2_grpc_read_db/read_db.proto

```diff
@@ -43,7 +43,7 @@ message AssociatedMessageType {
 
 message DbPullRequest {
   SourceId source_id = 1;
-  QueryId init_query_id = 2;
+  optional QueryId init_query_id = 2;
```

Comment:
This is proto3. All fields are optional here. I don' think we need to change this.
But if you want to change it you also need to update the gRPC version